hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/config/EvmConfiguration.java

```diff
@@ -0,0 +1,28 @@
+package com.hedera.mirror.web3.evm.config;
+
+import com.github.benmanes.caffeine.cache.Caffeine;
+import java.util.concurrent.TimeUnit;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.cache.CacheManager;
+import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.cache.caffeine.CaffeineCacheManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EntityScan("com.hedera.mirror.web3.evm")
+@EnableCaching
+public class EvmConfiguration {
+
+    @Bean
+    public Caffeine caffeineConfig() {
+        return Caffeine.newBuilder().expireAfterAccess(10, TimeUnit.MINUTES);
+    }
+
+    @Bean
```

Comment:
Since we'll probably have multiple different caches, recommend you provide it a distinct name and reference that name in the `@Cacheable`.
```suggestion
    @Bean(CACHE_MANAGER_10M)
```